products/coldfusion.md

```diff
@@ -18,32 +18,49 @@ releases:
     releaseDate: 2023-05-17
     eol: 2028-05-16
     extendedSupport: 2029-05-16
-    link: null # does not exist yet, will probably become valid after the first update
+    latestReleaseDate: 2023-10-05
+    latest: "ColdFusion (2023 Release) Update 5"
+    link: https://coldfusion.adobe.com/2023/05/coldfusion2023-release/
 
 -   releaseCycle: "2021"
     releaseDate: 2020-11-11
     eol: 2025-11-10
     extendedSupport: 2026-11-10
+    latestReleaseDate: 2023-10-06
+    latest: "ColdFusion (2021 release) Update 11"
```

Comment:
```suggestion
    latest: "ColdFusion (2021 release) Update 11"
    latestReleaseDate: 2023-10-06
```

`latest` and `latestReleaseDate` must be inverted, that's the usual formatting on endoflife.date.

Reply:
Merci! I just made another commit to insure that `latest `proceeds `latestReleaseDate`